tests/test_xarray.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import annotations
+
+import pytest
+import xarray as xr
+
+import coincident
+from coincident.io.xarray import plot_esa_worldcover, to_dataset
+
+# These tests are based on the workflow from https://coincident.readthedocs.io/en/latest/examples/contextual_data.html
+# I couldn't think of a way to generalized these tests since the
+# xarray funcs are pretty tailored to COP30 and ESA WC
+
+# mypy doesn't like the resolution argument in to_dataset
+# Argument "resolution" to "to_dataset" has incompatible type "float"; expected "dict[str, Any]"  [arg-type]
+
+@pytest.fixture
+def search_aoi():
+    """Fixture to load a simplified AOI from realistic lidar data."""
+    workunit = "CO_WestCentral_2019"
+    df_wesm = coincident.search.wesm.read_wesm_csv()
+    gf_lidar = coincident.search.wesm.load_by_fid(
+        df_wesm[df_wesm.workunit == workunit].index
+    )
+    return gf_lidar.simplify(0.01)
```

Comment:
This is great! Thanks for giving these patterns a go. While this definitely works, I suggest we try to minimize network traffic for network tests. As the library grows we'll want to reduce the total time required to run all the tests. 

Here you could instead use a smaller polygon. For example, consider moving this one into __init__.py so that it can be used in any of the test_ files

https://github.com/uw-cryo/coincident/blob/c5a1a8521fe853420c0434b330159d1171ca7231/tests/test_search.py#L25-L29